### PR TITLE
Create two_Unique_No.c

### PR DESCRIPTION
**Pull request by ashishlakhani04**
Added the code of two Unique Numbers.

**Fixes issue:** #[Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue.]

**Changes:**
[Add here what changes were made in this pull request.]

### CODE REVIEW

File: code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+int main(){
+    
+    int n,i;
+    int a[] = {1,3,5,6,3,2,1,2};
+    n = sizeof(a)/sizeof(int);
+    findUnique2(a,n);
+    
+    
+    return 0;
+
```
Comment (arnavb) [inline code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c]: Remove extra spaces below and and above `return 0;`

File: code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    int secondNo = res^firstNo;
+    printf("%d\n",firstNo);
+    printf("%d\n",secondNo);
+    
+}
+
+
+int main(){
+    
+    int n,i;
```
Comment (arnavb) [inline code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c]: What is `i`?

File: code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+void findUnique2(int *a,int n){
```
Comment (arnavb) [inline code/bit-manipulation/twice_unique_number/two_Unique_Numbers.c]: Add a space in between the include and function.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**